diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/FileNaming.kt

```diff
@@ -39,8 +39,10 @@ class FileNaming(private val configRules: List<RulesConfig>) : Rule("file-naming
 
         if (node.elementType == FILE) {
             filePath = node.getFilePath()
-            checkFileNaming(node)
-            checkClassNameMatchesWithFile(node)
+            if (!filePath.endsWith(".kts")) {
```

Comment:
You use it in several places and we may need it somewhere else, maybe make it lazy or smth else?

Reply:
I'll add an extension `isScript` or something similar.

Reply:
Done